_pages/en_US/include/soundhaxploit.txt

```diff
@@ -0,0 +1,62 @@
+### What You Need
+{%- if include.sp == "true" %}
+* The latest release of [Soundhax](http://soundhax.com) *(for your region and device)*
+  + for the version, use:
+  + 1.x - 2.1 if the number is between 0 and 2
```

Comment:
if what number

Reply:
the two numbers before the region in the system version string